### ISSUE TITLE
torch.nn.functional.sigmoid and torch.nn.functional.tanh deprecated

### ISSUE THREAD

**Issue by tsjayram** [IBM/mi-prometheus#142]
**Describe the bug**
When I use `torch.nn.functional.sigmoid` and `torch.nn.functional.tanh` I get warnings that they are deprecated and to use `torch.sigmoid` and `torch.tanh` instead. Apparently, this is because tensors and variables are now "merged".

**Desktop (please complete the following information):**
 - PyTorch version 1.0
